packages/common-types/src/node-protocol.ts

```diff
@@ -20,6 +20,7 @@ export namespace Node {
   // SOURCE: https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/API_REFERENCE.md#public-methods
   export enum MethodName {
     GET_APP_INSTANCES = "getAppInstances",
+    GET_PROPOSED_APP_INSTANCES = "getProposedAppInstances",
```

Comment:
does this PR need to add this to the list of public methods? I don't see it there currently

Reply:
I didn't see it there, so I added it.

Reply:
same comment for CREATE_MULTISIG and GET_CHANNEL_ADDRESSES

Reply:
as in, doesn't this PR need to edit `cf.js/API_REFERENCE.md` to keep it in sync with this change?